src/api/README.md

```diff
@@ -1642,6 +1642,126 @@ Result:
 }
 ```
 
+## Transaction Note APIs
+
+### Get all existing notes
+
+API sets: `READ`
+
+```
+URI: /api/v2/notes
+Method: GET
+```
+
+Gets all the existing notes to all of the transactions.
+
+Example:
+
+```sh
+curl http://127.0.0.1:6420/api/v2/notes
+```
+
+Result:
+
+```json
+{
+    "data": {
+        "a5cf149da9cab9fdff681cec9fe83983aada218a46e26292a2c977ceff5bb1a5": "note1",
+        "db6fec68266296fcf6bf98a26cf25d86c83bfc31b8248575724977d90426addd": "note2",
+        "fef07801a566c3eafd680c9d29ccc18657c600e8b9d8f2c0eb89e3c98f5019c4": "note3"
+    }
+}
+```
+
+### Get transaction note by transaction id
+
+API sets: `READ`
+
+```
+URI: /api/v2/note
```

Comment:
@gz-c is it possible for URL to be `/api/v2/note/{txid}` e.g. `http://127.0.0.1:6420/api/v2/note/a5cf149da9cab9fdff681cec9fe83983aada218a46e26292a2c977ceff5bb1a5` ?

Reply:
I was thinking it could support other types too (uxid, address).  Maybe this should be combined into the general key-value store and let the application put any string they want.

Reply:
@olemis we don't follow the `/path/to/{var}` convention for the other endpoints so we won't do it here